### DISCUSSION TITLE
Multiple Reconcilation Errors

### DISCUSSION THREAD

**Discussion by jjvasquezv** [aquasecurity/trivy#7413] [Adopters]
I am trying to deploy the trivy operator on a kubernetes cluster in Ms Azure AKS. The installation has been executed using kubectl with the following command: 

kubectl apply -f https://raw.githubusercontent.com/aquasecurity/trivy-operator/v0.22.0/deploy/static/trivy-operator.yaml

As a result a new namespace has been created and the deployment, pod and containers have been successfully created and started. However, the reports have not been generated and there are multiple reconciliation errors in the deployment. 

Ex:
"Reconciler error","controller":"node","controllerGroup":"","controllerKind":"Node","Node":{"name":"aks-nodepool1-19115329-vmss00000i"},"namespace":"","name":"aks-nodepool1-19115329-vmss00000i","reconcileID":"27fa2053-2b1d-499b-a3a2-ae9d77862766","error":"creating job: no compliance commands found","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.18.4/pkg/internal/controller/controller.go:324\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.18.4/pkg/internal/controller/controller.go:261\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.18.4/pkg/internal/controller/controller.go:222"}

Any suggestion? I'm just following the trivy operator documentation at https://aquasecurity.github.io/trivy-operator/latest/getting-started/installation/kubectl/

**Comment by afdesk** [comment]
@jjvasquezv thanks for the report. I'll take a look at this issue

**Comment by afdesk** [comment]
@jjvasquezv it looks like there is the same problem in `trivy-operator` - https://github.com/aquasecurity/trivy-operator/issues/2227
if I'm wrong and you expirience a new issue, could you arise a ticket there? 
thanks again!

**Reply by lightln2** [reply]
Are you sure it's the same problem? the error in the ticket you refer to, is "illegal base64 data at input byte 17200", controller=job, this error says "creating job: no compliance commands found", node controller

**Reply by afdesk** [reply]
@lightln2 i'm not sure. I just suggested it's a ticket for `trivy-operator`, not for `trivy`.
or do i miss something? thanks
